Extensions/DeviceSensors/devicesensortools.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * @memberof gdjs.evtTools
+ * @class devicesensors
+ * @static
+ * @private
+ */
+
+gdjs.evtTools.devicesensors = {};
+gdjs.evtTools.devicesensors.orientation = {};
+
+gdjs.evtTools.devicesensors.orientation._isActive = false;
+
+function _activateOrientationListener() {
+  window.addEventListener("deviceorientation", _handleOrientation, true);
+  gdjs.evtTools.devicesensors.orientation._isActive = true;
+}
+
+function _deactivateOrientationListener() {
+  window.removeEventListener('deviceorientation', _handleOrientation, true);
+  gdjs.evtTools.devicesensors.orientation._isActive = false;
+}
+
+function _handleOrientation(event) {
```

Comment:
Ah good to know, I always thought it was the other way around. (anything declared inside the extension's top level would be local to that extension and stuff attached to evtTools would be accessable from the outside)

Reply:
@Wend1go It's the case when you're using Node.js (or bundler like Webpack, Parcel.js) but GDevelop is not using any kind of bundler and relying instead on "Vanilla" JavaScript (for simplicity sake, including a bundler is not an easy task, and also because it would slow down preview and exports).

So all code is living in the global scope, hence the need to scope everything to an object/module to avoid any name clashes.